db/migrate/20200901082951_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :body
+      t.integer :author_id
```

Comment:
- [ ] Change this to reflect the new user table as `user_id`, by making a new migration.

Reply:
It's already user_id. I believe you are looking at an old commit.